.tool-versions

```diff
@@ -0,0 +1 @@
+golang 1.17.13
```

Comment:
I'll upgrade Go in a follow-up PR, I don't want to change too many things at once